regulations/templates/regulations/preamble-base.html

```diff
@@ -22,7 +22,11 @@
 {% block sub-head-class %}preamble-sub-head{% endblock %}
 
 {% block active_title %}
-  <em class="header-label">{{section_label}}</em>
+  <em class="header-label">
+    {% if section_label != None %}
```

Comment:
Ah, good idea to check the view. I think it makes sense to set it to `''` if there isn't one to avoid logic in the template.